lib/naive_api/schema/types.ex

```diff
@@ -69,14 +87,14 @@ defmodule NaiveApi.Schema.Types do
   @desc "Room invite content"
   object :room_invite_content do
     field(:room_card, :card |> non_null)
-    field(:invitation, :bitstring |> non_null)
+    field(:invitation, :string |> non_null)
   end
 
   @desc "File content"
   object :file_content do
     field(:url, :string |> non_null)
     field(:type, :file_content_type |> non_null)
-    field(:size_bytes, :integer |> non_null)
+    field(:size, :string |> non_null)
```

Comment:
Why to convert to string ? 
it is API client duty to convert into human readable if needed